pkgs/by-name/bs/bsdgames/package.nix

```diff
@@ -27,13 +28,44 @@ stdenv.mkDerivation rec {
   ];
 
   patches = [
-    # Remove UTMPX support on Makefrag file
-    (fetchurl {
+    # Follow All patches from http://t2sde.org/packages/bsd-games.html
+    # May be removed in the next version
+    (fetchpatch {
+      url = "http://svn.exactcode.de/t2/trunk/package/games/bsd-games/delay_output-sym.patch";
+      hash = "sha256-qbtp2cJVmEuxbTgpXM2gRHSNLE25OCmNxK+aeBBPRBw=";
+    })
+    (fetchpatch {
       url = "http://svn.exactcode.de/t2/trunk/package/games/bsd-games/dm-noutmpx.patch";
-      sha256 = "1k3qp3jj0dksjr4dnppv6dvkwslrgk9c7p2n9vipqildpxgqp7w2";
+      hash = "sha256-oioJ5M7DAUVzX8k998p2h/APn8azw9Z8txmBuly2ouw=";
+    })
+    (fetchpatch {
+      url = "http://svn.exactcode.de/t2/trunk/package/games/bsd-games/group-games.patch";
```

Comment:
Is this thing relevant to us? Our packages cannot set groups, and a module would be another story.

Reply:
But the patch is only relevant in a context completely different from ours, we can just skip it.